packages/now-client/package.json

```
@@ -1,10 +1,11 @@
 {
   "name": "now-client",
-  "version": "5.1.1-canary.1",
-  "main": "lib/index.js",
+  "version": "5.1.1-canary.0",
```

```suggestion
  "version": "5.1.1-canary.1",
```

We can't go backwards in numbers or else lerna might not publish properly